english/hammertime/hammertime.md

```diff
@@ -5,6 +5,8 @@ Stop......IT'S HAMMERTIME - WHOOA-O, WHOOA-O! WHOOA-O, WHOOA-O!
 
 [Except when it is not](https://www.youtube.com/watch?v=dKv6Yco0Lzg)
 
-Then you see Lewis Hamilton speeding past on his last lap on those 
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.